recipes/pygeohash/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pygeohash" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 750e51643e743eabd065a84fc8c1912c5843b648143137919e4a9776366d921e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  files:
+    - tests/test_pygeohash.py
+  imports:
+    - pygeohash
+
+about:
+  home: https://github.com/wdm0006/pygeohash
+  license: AGPL-3.0
+  license_family: AGPL
+  license_file: LICENSE.txt
+  summary: 'A module for decoding and encoding geohashes to and from latitude and longitude coordinates'
+
+  # The remaining entries in this section are optional, but recommended.
```

Comment:
This should be removed

Reply:
@chrisburr are you referring to the comment section? Will do that right away